src/thread/id.rs

```diff
@@ -15,3 +15,45 @@ use crate::process::Pid;
 pub fn gettid() -> Pid {
     backend::thread::syscalls::gettid()
 }
+
+/// `setuid(uid)`—Sets the effective user ID of the calling process.
```

Comment:
Since this isn't the same as POSIX `setuid`, I think it makes sense to omit the code-quoted part here.